src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/DefaultRazorProjectHost.cs

```diff
@@ -104,8 +104,15 @@ internal async Task OnProjectChanged(IProjectVersionedValue<IProjectSubscription
                     if (TryGetConfiguration(update.Value.CurrentState, out var configuration))
                     {
                         TryGetRootNamespace(update.Value.CurrentState, out var rootNamespace);
+
                         var hostProject = new HostProject(CommonServices.UnconfiguredProject.FullPath, configuration, rootNamespace);
 
+                        if (TryGetIntermediateOutputPath(update.Value.CurrentState, out var intermediatePath))
```

Comment:
We need this logic to also run for our `FallbackRazorProjectHost`. To do that you can see if you can move this to the shared base type or replicate the logic.

For context:
We added the Razor SDK in .NET Core 2.1 and that gave us the ability to analyze a project using Razor SDK's MSBuild targets, properties, item groups etc. and extract things like the important Razor files, Razor configuration, language version etc.

Upon the Razor SDKs creation we built this `DefaultRazorProjectHost`. It's purpose was to analyze >= 2.1 projects and construct a project understanding of the world based on the Razor SDKs outputs (Razor files, lang version etc.).

However, given that we created the Razor SDK in 2.1 we couldn't just let < 2.1 projects falter; that's where `FallbackRazorProjectHost` comes in. The `FallbackRazorProjectHost` uses a set of heuristics to determine the set of important Razor files, Razor configuration, language version etc. without having a Razor SDK.